httpapi/v2/documents_api.go

```diff
@@ -72,3 +74,49 @@ func (h handler) CreateDocument(w http.ResponseWriter, r *http.Request) {
 	render.Status(r, http.StatusCreated)
 	render.JSON(w, r, resp)
 }
+
+// Commit creates a document.
+// @summary Commits a pending document.
+// @description Commits a pending document.
+// @id commit_document_v2
+// @tags Documents
+// @accept json
+// @param authorization header string true "Hex encoded centrifuge ID of the account for the intended API action"
+// @param document_id path string true "Document Identifier"
+// @produce json
+// @Failure 400 {object} httputils.HTTPError
+// @Failure 500 {object} httputils.HTTPError
+// @Failure 403 {object} httputils.HTTPError
+// @success 201 {object} coreapi.DocumentResponse
```

Comment:
the status should be accepted 202. WDYT ?

Reply:
yes!